add_new_dc_test.py

```diff
@@ -0,0 +1,120 @@
+import warnings
+from typing import Tuple, List
+
+from cassandra import ConsistencyLevel
+from cassandra.query import SimpleStatement  # pylint: disable=no-name-in-module
+
+from longevity_test import LongevityTest
+from sdcm.cluster import BaseNode
+from sdcm.stress_thread import CassandraStressThread
+from sdcm.utils.replication_strategy_utils import NetworkTopologyReplicationStrategy
+
+warnings.filterwarnings(action="ignore", message="unclosed", category=ResourceWarning)
+
+
+class TestAddNewDc(LongevityTest):
+    """Test for procedure:
+     https://docs.scylladb.com/operating-scylla/procedures/cluster-management/add-dc-to-existing-dc/
+     """
+
+    def test_add_new_dc(self) -> None:  # pylint: disable=too-many-locals
+
+        assert self.params.get('n_db_nodes').endswith(" 0"), "n_db_nodes must be a list and last dc must equal 0"
```

Comment:
why? you could have 2 DCs, and this test will add a 3rd one

Reply:
adding another dc with 0 makes everything much easier. Without it I'd need to mess with undefined number of things to setup by myself in the code, just to make sct happy (e.g. _ec2_subnet_id, _ec2_security_group_ids, _ec2_ami_id, credentials and possibly many more).

Reply:
The more interesting scenario is growing from 1 DC to 2 DCs.

Reply:
I was trying to follow Trello task - where is written we should start with 2. 
I'd be happy to change it back to scenario 1DC->2DC.